drivers/storage/portworx/manifest/manifest.go

```diff
@@ -293,6 +294,7 @@ func fillCSIDefaults(
 	rel.Components.CSISnapshotter = csiImages.Snapshotter
 	rel.Components.CSISnapshotController = csiImages.SnapshotController
 	rel.Components.CSIHealthMonitorController = csiImages.HealthMonitorController
+	rel.Components.CsiLivenessProbe = k8sutil.DefaultK8SRegistryPath + "/sig-storage/livenessprobe:v2.7.0"
```

Comment:
Ideally all these csi default images should come from the csiImages object above. We should add them to the csi_generator.go to keep them in one place.

Reply:
done